model/model_training/custom_datasets/qa_datasets.py

```diff
@@ -343,16 +343,16 @@ def __init__(self, cache_dir) -> None:
                 # DO NOT change this
                 # its the data that had syntax error
                 explanation = data["explaination"]
-                self.pairs.append((joke, explanation))
+                self.pairs.append(DatasetEntry([joke], [explanation]))
```

Comment:
you rely here on the order of the keywords. Could we make this explicit by using
```suggestion
                self.pairs.append(DatasetEntry(questions=[joke], answers=[explanation]))
```

Reply:
Removed the redundant code.